docs/projects/version-control/converting/index.md

```diff
@@ -1,34 +1,35 @@
 ---
-title: Configuring version control on a Project
-description: Configuring a project to leverage the configuration as a code feature. 
+title: Converting projects to Git
+description: Converting a project to leverage the configuration as a code feature.
 position: 10 
 ---
 
-Version-control is configured per project and is accessed via the **{{ Settings, Version Control }}** link in the project navigation menu. This page will walk through how to configure a project to be version-controlled.
+Git settings are configured per project and are accessed via the **{{ Settings, Version Control }}** link in the project navigation menu. This page will walk through how to convert a project to Git.
 
 ![Version-control configuration UI](version-control-configuration.png "width=500")
 
 ## Creating a new version-controlled project
 
-To get a feel for the config-as-code feature, you may want to create a new project that you can test with before committing to permanently converting an existing project. This project's deployment process will be stored in a git repository when configured.
+To get a feel for the config-as-code feature, you may want to create a new project that you can test with before committing to permanently converting an existing project. This project's deployment process, deployment settings, and variables will be stored in a Git repository when configured.
```

Comment:
```suggestion
To get a feel for the config-as-code feature, it is recommended that a new project is created that can be tested before committing to permanently converting an existing project. This project's deployment process, deployment settings, and variables will be stored in a Git repository when configured.
```

Reply:
I'd like to keep this one as active as possible, and keep it as an optional task (not necessarily a recommended one).